src/impl_x86__base_implementation.inl

```diff
@@ -1799,7 +1801,7 @@ CacheInfo GetX86CacheInfo(void) {
   LINE(AMD_HAMMER)                  \
   LINE(AMD_K10)                     \
   LINE(AMD_K11)                     \
-  LINE(AMD_K12)                     \
+  LINE(AMD_K12) /* K12 LLANO */     \
```

Comment:
@gchatelet, did you mean to add a comment here or `include/cpuinfo_x86.h -> X86Microarchitecture`?

Reply:
The latter `include/cpuinfo_x86.h`